### PR TITLE
[ACVOD-137]: Fix subtitle position

### PR DESCRIPTION
**Pull request by Yarrow228**
Subtitle position in videoplayer

### CODE REVIEW

File: src/app/pages/video-player/video-player.component.ts [hunk 1]
```diff
@@ -178,4 +179,24 @@ export class VideoPlayerComponent implements OnInit, AfterViewInit {
         });
     });
   }
+
+  showSubtitles() {
+    const videoElement = document.querySelector('video').textTracks[
```
Comment (Semenone) [inline src/app/pages/video-player/video-player.component.ts:184]: why you use querySelector instead viewChild(viewChldren)?

File: src/app/pages/video-player/video-player.component.ts [hunk 2]
```diff
@@ -24,16 +24,18 @@ export class VideoPlayerComponent implements OnInit, AfterViewInit {
...
 
   public routeId;
   public routeType: string;
   public currentTvShowId;
   public currentSeasonId;
-  public isMouseMove = false;
-  public timeout: NodeJS.Timeout;
-  public nextVideo;
   public currentVideo;
+  public nextVideo;
```
Comment (marinbonibon) [inline src/app/pages/video-player/video-player.component.ts:34]: What types does these properties have?

File: src/app/pages/video-player/video-player.component.ts [hunk 3]
```diff
@@ -178,4 +180,25 @@ export class VideoPlayerComponent implements OnInit, AfterViewInit {
...
+      subtitleElement.textContent = '';
+    } else {
+      videoElement.oncuechange = function (e) {
+        let cue = this.activeCues[0];
+        if (cue) {
+          subtitleElement.textContent = cue.text;
+        } else {
+          subtitleElement.textContent = '';
+        }
+      };
```
Comment (marinbonibon) [inline src/app/pages/video-player/video-player.component.ts:201]: Please, refactor this part in order to not repeat subtitleElement.textContent = '' and not have a lot of if else. And use arrow functions. And why do you need e if you don't use it? And why let?

**Review by Semenone — APPROVED** [review]

File: src/app/pages/video-player/video-player.component.ts [hunk 4]
```diff
@@ -182,23 +182,21 @@ export class VideoPlayerComponent implements OnInit, AfterViewInit {
...
-          subtitleElement.textContent = cue.text;
-        } else {
-          subtitleElement.textContent = '';
-        }
-      };
+      subtitleTrack.addEventListener('cuechange', (event) => {
+        const cue = event.target.activeCues[0];
+        cue
+          ? (subtitleElement.textContent = cue.text)
+          : (subtitleElement.textContent = '');
```
Comment (marinbonibon) [inline src/app/pages/video-player/video-player.component.ts:197]: I meant to change this subtitleTrackIndex === -1 condition to the opposite and make subtitleElement.textContent = '' default, and  videoElement.oncuechange was ok) And here if you want to use ternary than `subtitleElement.textContent = cue ?  cue.text : ''`;

**Pull request merged**